presets/consumer/avformat/Screencast-H264

```diff
@@ -0,0 +1,23 @@
+pix_fmt=yuv420p
+vcodec=libx264
+vprofile=high
+level=51
+crf=27
+g=249
+bf=8
+preset=veryfast
+b-pyramid=0
+refs=8
+weightb=1
+b_strategy=2
+rc-lookahead=25
+channels=1
```

Comment:
I think it is bad to include channels because in Shotcut that Export parameter defaults from its Settings > Audio Channels. Choosing this preset could surprise people who want stereo.

Reply:
Hmm, I'm torn on this one.  These export presets are extremely specialized for media types that are 99% mono human speech.  Most video conferencing programs only capture mono audio.  Adding another (likely identical) channel of audio increases file size by 25% which defeats the project charter of "smallest file size possible" for the expected use cases.  I am not at all opposed to removing the `channels=1` flag if that's the convention, but I'm guessing more users could be surprised by larger file size than would be surprised by mono audio.  I defer to you.

Reply:
A very large portion of Shotcut users are people recording games. I think a lot of them will see "Screencast" and think: oh, that is just like a game recording. These people will lose stereo output (not to mention poor audio quality due to the low bitrate).

Reply:
Yeah, I can see the confusion.  The original name I gave these presets was "Slide Deck (H.264)" before switching to "Screencast" at the last minute.  Perhaps "Slide Deck" remains more representative.  Granted, these presets are a bit niche, so I'm okay with retracting them if they will cause more confusion than benefit.  I was specifically targeting video conference and distance education users with COVID stay-at-home arrangements, hence the low bitrate audio that's sufficient for human speech.

Reply:
I like the idea of having these presets because I think the use case is common. Maybe the name needs some work. "Slide Deck" is not to bad. Other suggestions: "Presentation", "Slideshow", "High Compression".

Reply:
I agree to "Slide Deck" and then keep `channels=1`

Reply:
The name has been changed to "Slide Deck".  Thanks guys for your patience with me.

The output of these presets is around 10% the size of the YouTube preset.  For recordings of 2-hour class lectures and business meetings, the savings quickly become significant.